scripts/prepare_automatic_backups.py

```diff
@@ -34,6 +34,16 @@
 
 
 def generate_backup_url(cloud_storage_bucket_name, module_class_names):
+    """Generates backup url of new added urls in cron file.
```

Comment:
Why are we talking about "newly added" here? From what I can tell, the URL is generated directly from the names that are passed in, so I don't understand where the concept of "newly added" comes from.

Make sure that your docstring precisely describes what the function is doing.